js/src/forum/components/RenameDiscussionModal.js

```diff
@@ -58,6 +58,7 @@ export default class RenameDiscussionModal extends Modal {
           app.current.stream.update();
         }
         m.redraw();
+        app.setTitle(title());
```

Comment:
Hmm, this feels wrong in terms of the virtual DOM / always re-render model. Shouldn't this title be set whenever the discussion page is re-rendered? (This is just one of the places where we trigger a redraw of that page - that redraw in turn should take care of the title change.)

Reply:
That's true, however if it did update this PR wouldn't exist

Reply:
Looks like the title is currently only set on https://github.com/flarum/core/blob/master/js/src/forum/components/DiscussionPage.js#L163 (DiscussionPage#show), which only runs when DiscussionPage#refresh runs, aka on first load of Discussion, I assume.